cmd/utils/flags.go

```diff
@@ -117,6 +117,15 @@ var (
 		Name:  "directbroadcast",
 		Usage: "Enable directly broadcast mined block to all peers",
 	}
+	DisableSnapProtocolFlag = cli.BoolFlag{
+		Name:  "disablesnapprotocol",
+		Usage: "Disable snap protocol",
+	}
+	DiffSyncFlag = cli.BoolFlag{
+		Name: "diffsync",
+		Usage: "Enable difflayer sync, Please note that enable diffsync will improve the syncing speed, " +
```

Comment:
can we change difflayer to `diff layer`

Reply:
change `difflayer sync` into `diff sync`